src/Form/Form.js

```diff
@@ -144,7 +199,11 @@ class Form extends Component {
         if (this.state.usersValidationState === 'success' &&
             this.state.postTitle.validationState === 'success' &&
             this.state.postBody.validationState === 'success') {
-                this.__fetchApi(url, method, postData);
+                // this.__fetchApi(url, method, postData);
```

Comment:
comment to `if` above: maybe it is good idea to store global status 'success/error' for all fields generally to avoid such long if's. What if we will have 20 fields?